services/models/workload.go

```diff
@@ -36,4 +41,5 @@ func (workloadList *WorkloadList) Parse(namespace string, ds *v1beta1.Deployment
 
 func (workload *WorkloadOverview) Parse(d v1beta1.Deployment) {
 	workload.Name = d.Name
+	workload.Type = "Deployment"
 }
```

Comment:
The param in the future should be a interface of types : Deployment Replicaset ... Not Block now